### PR TITLE
Admin doc update

### PR DESCRIPTION
**Pull request by raymond-zhang00**
*Issue #, if available:*

*Description of changes:*
Updates to admin machine doc from doc review.

Incorporates and adds to changes from this PR:
https://github.com/aws/eks-anywhere/pull/6534/files

*Testing (if applicable):*

*Documentation added/planned (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/content/en/docs/getting-started/install/_index.md [hunk 1]
```diff
@@ -5,36 +5,38 @@ description: >
   Steps for setting up the Admin Machine
 ---
 
+{{% alert title="Warning" color="warning" %}}
+The Administrative machine should not be ephemeral. Artifacts created during cluster operations will be stored on the Administrative machine. Losing this machine could result in loss of cluster information.
```
Comment (csplinter) [inline docs/content/en/docs/getting-started/install/_index.md:9]: We can be clearer here. 
"The Administrative machine (Admin machine) is required to run cluster lifecycle operations, but EKS Anywhere clusters do not require a continuously running Admin machine to function. During cluster creation, critical cluster artifacts including the kubeconfig file, SSH keys, and the full cluster specification yaml are saved to the Admin machine. These files are required when running any subsequent cluster lifecycle operations. For this reason, it is recommended to save a backup of these files and to use the same Admin machine for all subsequent cluster lifecycle operations."

File: docs/content/en/docs/getting-started/install/_index.md [hunk 2]
```diff
@@ -5,36 +5,38 @@ description: >
...
+{{% /alert %}}
+
 EKS Anywhere will create and manage Kubernetes clusters on multiple providers.
 Currently we support creating development clusters locally using Docker and production clusters from providers listed on the [providers]({{< relref "/docs/getting-started/chooseprovider/" >}}) page.
 
-Creating an EKS Anywhere cluster begins with setting up an Administrative machine where you will run Docker and add some binaries.
-From there, you create the cluster for your chosen provider.
+Creating an EKS Anywhere cluster begins with setting up an Administrative machine where you will run all EKS Anywhere lifecycle operations as well as Docker, `kubectl` and the prerequisite utilites.
```
Comment (csplinter) [inline docs/content/en/docs/getting-started/install/_index.md:15]: Nit: change to present tense. "Creating an EKS Anywhere cluster beings with setting up an Admin machine where you run EKS Anywhere cluster lifecycle operations ..."

File: docs/content/en/docs/getting-started/install/_index.md [hunk 3]
```diff
@@ -5,36 +5,38 @@ description: >
...
+From here you will need to install [`eksctl`](https://eksctl.io), a CLI tool for creating and managing clusters on EKS, and the [`eksctl-anywhere`](/docs/reference/eksctl/anywhere/) plugin, an extension to create and manage EKS Anywhere clusters on-premises, on your Administrative machine.
+You can then create a cluster for your chosen provider.
```
Comment (csplinter) [inline docs/content/en/docs/getting-started/install/_index.md:17]: There are other steps before users can create a cluster (set up infra / environment, create OS image if using Ubuntu or RHEL, review cluster networking requirements)

File: docs/content/en/docs/getting-started/install/_index.md [hunk 4]
```diff
@@ -5,36 +5,38 @@ description: >
...
-This will let you create a cluster in multiple providers for local development or production workloads.
-
->**_NOTE:_** For Snow provider, the Snow devices will come with a pre-configured Admin AMI which can be used to create an Admin instance with all the necessary binaries, dependencies and artifacts to create an EKS Anywhere cluster. Skip the below steps and see [Create Snow production cluster]({{< relref "../snow" >}}) to get started with EKS Anywhere on Snow.
+>**_NOTE:_** For Snow provider, the Snow devices will come with a pre-configured Admin AMI which can be used to create an Admin instance with all the necessary binaries, dependencies and artifacts to create an EKS Anywhere cluster. Skip to the steps on [Create Snow production cluster]({{< relref "../snow/snow-getstarted" >}})to get started with EKS Anywhere on Snow.
 
 ### Administrative machine prerequisites
 
 #### System and network requirements
-- Mac OS 10.15 / Ubuntu 20.04.2 LTS
+- Mac OS 10.15+ / Ubuntu 20.04.2 LTS or 22.04 LTS
```
Comment (csplinter) [inline docs/content/en/docs/getting-started/install/_index.md:25]: @jaxesn do we want to add Rocky Linux and RHEL?

File: docs/content/en/docs/getting-started/install/_index.md [hunk 5]
```diff
@@ -5,36 +5,38 @@ description: >
...
+You can then create a cluster for your chosen provider. 
 See [Create cluster workflow]({{< relref "../overview" >}}) for an overview of the cluster creation process.
 
-To create an EKS Anywhere cluster you will need [`eksctl`](https://eksctl.io) and the `eksctl-anywhere` plugin.
-This will let you create a cluster in multiple providers for local development or production workloads.
-
->**_NOTE:_** For Snow provider, the Snow devices will come with a pre-configured Admin AMI which can be used to create an Admin instance with all the necessary binaries, dependencies and artifacts to create an EKS Anywhere cluster. Skip the below steps and see [Create Snow production cluster]({{< relref "../snow" >}}) to get started with EKS Anywhere on Snow.
+>**_NOTE:_** For Snow provider, the Snow devices will come with a pre-configured Admin AMI which can be used to create an Admin instance with all the necessary binaries, dependencies and artifacts to create an EKS Anywhere cluster. Skip to the steps on [Create Snow production cluster]({{< relref "../snow/snow-getstarted" >}})to get started with EKS Anywhere on Snow.
```
Comment (csplinter) [inline docs/content/en/docs/getting-started/install/_index.md:20]: "If you ordered a Snowball Edge device with EKS Anywhere enabled, it is preconfigured with an Admin AMI which contains the necessary binaries, dependencies, and artifacts to create an EKS Anywhere cluster. Skip to the steps on [Create Snow production cluster]({{< relref "../snow/snow-getstarted" >}})to get started with EKS Anywhere on Snow."

File: docs/content/en/docs/getting-started/install/_index.md [hunk 6]
```diff
@@ -5,36 +5,38 @@ description: >
...
+- [Docker 20.x.x](https://docs.docker.com/engine/install/)
+- [`curl`](https://everything.curl.dev/get)
+- [`yq`](https://github.com/mikefarah/yq/#install)
 
-If you are using Ubuntu, use the Docker CE installation instructions to install Docker and not the Snap installation, as described [here.](https://docs.docker.com/engine/install/ubuntu/)
-
-* For EKS Anywhere Bare Metal, Docker Desktop is not supported
+{{% alert title="Note" color="primary" %}}
+- If you are using Ubuntu, use the Docker CE installation instructions to install Docker as described [here.](https://docs.docker.com/engine/install/ubuntu/) 
+- For EKS Anywhere Bare Metal, Docker Desktop is not supported.
```
Comment (csplinter) [inline docs/content/en/docs/getting-started/install/_index.md:38]: EKS Anywhere on bare metal

File: docs/content/en/docs/getting-started/install/_index.md [hunk 7]
```diff
@@ -106,59 +108,7 @@ eksctl anywhere version
...
-
-### For Bare Metal (Tinkerbell)
-You will need to have hookOS and its OS artifacts downloaded and served locally from an HTTP file server.
-You will also need to modify the [hookImagesURLPath]({{< relref "../baremetal/bare-spec/#hookimagesurlpath" >}}) and the [osImageURL]({{< relref "../baremetal/bare-spec/#osimageurl" >}}) in the cluster configuration files.
-Ensure that structure of the files is set up as described in [hookImagesURLPath.]({{< relref "../baremetal/bare-spec/#hookimagesurlpath" >}})
-
-### For vSphere
-If you are using the vSphere provider, be sure that the requirements in the
-[Prerequisite checklist]({{< relref "../vsphere/vsphere-prereq" >}}) have been met.
+For more information on how to prepare the Administrative machine for airgapped environments, go to the [Airgapped](/docs/getting-started/airgapped/) page.
```
Comment (chrisnegus) [inline docs/content/en/docs/getting-started/install/_index.md:111]: Thanks for removing this sections. That should have been done when we did the reorg a while back.

File: docs/content/en/docs/getting-started/install/_index.md (hunk 6 above)
Comment (chrisnegus) [reply docs/content/en/docs/getting-started/install/_index.md:38]: We have been treating "Bare Metal" as a sort of product name. If you don't want that, we should go back and change it in other places as well.

**Review by chrisnegus — COMMENTED** [review]
Just a couple of comments.

File: docs/content/en/docs/getting-started/install/_index.md (hunk 1 above)
Comment (raymond-zhang00) [reply docs/content/en/docs/getting-started/install/_index.md]: Sounds good, using updated verbiage.

File: docs/content/en/docs/getting-started/install/_index.md (hunk 2 above)
Comment (raymond-zhang00) [reply docs/content/en/docs/getting-started/install/_index.md]: Updated tense.

File: docs/content/en/docs/getting-started/install/_index.md (hunk 5 above)
Comment (raymond-zhang00) [reply docs/content/en/docs/getting-started/install/_index.md]: Added this after 'For Snow Provider'

File: docs/content/en/docs/getting-started/install/_index.md (hunk 4 above)
Comment (jaxesn) [reply docs/content/en/docs/getting-started/install/_index.md:25]: Yeah lets do it

Comment (raymond-zhang00) [reply docs/content/en/docs/getting-started/install/_index.md]: Confirmed and updated to 8.8+

File: docs/content/en/docs/getting-started/install/_index.md (hunk 6 above)
Comment (raymond-zhang00) [reply docs/content/en/docs/getting-started/install/_index.md:38]: Does EKS Anywhere Bare Metal work for the time being to match our other documentation?

Comment (csplinter) [reply docs/content/en/docs/getting-started/install/_index.md:38]: yea that's fine

**Review by csplinter — APPROVED** [review]

**Pull request merged**